src/components/Sensitivity/Sensitivity.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Chip, makeStyles, Typography } from '@material-ui/core';
+import { VerifiedUserOutlined } from '@material-ui/icons';
+import { FC } from 'react';
+import * as React from 'react';
+import { Sensitivity as SensitivityType } from '../../api';
+
+const useStyles = makeStyles(
+  ({ palette: { secondary, warning }, spacing }) => ({
+    icon: {
+      height: spacing(2),
+      width: spacing(2),
+      marginRight: spacing(0.5),
+    },
+    iconWrapper: {
+      display: 'flex',
+      alignItems: 'center',
+      marginBottom: spacing(),
+    },
+    chip: {
+      borderRadius: '4px',
+      color: 'white',
+      backgroundColor: ({ value }) => {
+        if (value === 'High') {
+          return secondary.main;
+        }
+
+        if (value === 'Medium') {
+          return warning.main;
+        }
+
+        return '#8f928b';
```

Comment:
This felt a little weird to add this color here. I need this light grey color for when the button says `Low` - is there a best practice to pull this in from the theme? It's used in a different spot, too, so I made a variable but couldn't quite pin down how to reference it. Any guidance is appreciated here!

Reply:
This is use case is a little tricky.
An argument could be made that these colors should just be defined here and not pulled from the theme. For example, we don't want to couple _High_ to our `secondary` color palette. That color could change and that doesn't mean we'd want this usage to change.
However, we also have `error` and `warning` definitions as well, which probably do mean the same thing as what we are trying to communicate here.

So it's more about mapping the intention of the theme option to the usage, rather than the color. If we can find a gray that makes sense in the theme and can be changed to the gray in designs and it still work across the app then let's do that. Otherwise we can just hardcode it here.

I'll dig more.

Reply:
10-4